Compiler/runtime/systemimpl.c

```diff
@@ -2230,6 +2230,7 @@ extern char* SystemImpl__iconv(const char * str, const char *from, const char *t
   iconv_t ic;
   int count;
   char *buf;
+  if (!strcmp(from, to)) return (char*)str;
```

Comment:
Well, we could do that check always since usually things are ASCII 7-bit and won't need the allocated memory. It's also a bit of a problem since the external C interface will always make a copy of the string anyway (`external "MetaModelica"` would be nice to have).